pyaedt/emit.py

```diff
@@ -34,45 +11,64 @@
 
 
 class Emit(FieldAnalysisEmit, object):
-    """Emit Object. This is an alpha implementation of Emit object.
-    It contains only skeleton to create empty design.
-    No methods implemented at this stage
+    """Emit class to that exposes Emit functionalities.
```

Comment:
```suggestion
    """Emit class.
```

Reply:
Again, we need better descriptions.  Mine short descriptions are pretty crappy...

Reply:
Not sure what to change it to since I don't think there's an application interface to Emit.  I'll commit your suggestion as it as it's better than nothing and at least sounds like english.